pkg/setting/setting_plugins.go

```diff
@@ -96,10 +96,19 @@ func (cfg *Cfg) readPluginSettings(iniFile *ini.File) error {
 		for _, plugin := range preinstallPlugins {
 			cfg.PreinstallPlugins = append(cfg.PreinstallPlugins, plugin)
```

Comment:
nit: Can we rename this to `cfg.PreinstallPluginsAsync`?  With that naming, it's clear that there are two lists and what differentiates them.

Reply:
I thought about that, but we still have flag variable `PreinstallPluginsAsync` in the config

Reply:
right, which is even more confusing since that is not a list. I don't think we actually need to store the value of the flag in the config anymore since by the time the settings are read, we have the two final lists of sync / async plugins so we can clean this up making `cfg.PreinstallPluginsAsync` the list of async installations and removing `cfg.PreinstallPlugins`

Reply:
@andresmgot ready for review.